include/etiss/DebugSystem.h

```diff
@@ -76,21 +150,35 @@ class DebugSystem : public System
             @brief loads a binary image from a file to the given address
     */
     // bool load(etiss::uint64 addr,const char * file);
-    bool loadRom(const char *file);
```

Comment:
why were these removed? we still provide the interface and access functions for the ram/rom sections, but without these we cannot initialize the data